llvm/test/CodeGen/AMDGPU/no-corresponding-integer-type.ll

```diff
@@ -0,0 +1,13 @@
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx942 %s -o -
+
+define void @no_corresponding_integer_type(i8 %arg, ptr addrspace(1) %ptr) {
```

Comment:
I realise this is a crash test, but its still nice to use FileCheck and check the codegen